modin_spreadsheet/tests/test_grid.py

```diff
@@ -948,3 +949,32 @@ def test_apply_history():
 
     # Checks that the history is applied properly
     assert changed_df.equals(applied_df)
+
+
+def test_mixed_type_sort():
+    df = pd.DataFrame(
+        {
+            "Mixed": [pd.Timestamp("2017-02-02"), np.nan, 1e10, 5, "hey"],
+            1: [5, 4, "hello", 2, 1],
+        }
+    )
+
+    view = SpreadsheetWidget(df=df)
```

Comment:
Let's use the variable name `spreadsheet` instead of `view`. Make corresponding changes throughout test.
```suggestion
    spreadsheet = SpreadsheetWidget(df=df)
```

Reply:
Done!